package-lock.json

```diff
@@ -0,0 +1,5893 @@
+{
```

Comment:
NPM 5's `package-lock.json` is currently (unless I've missed something) a bit 'up in the air' (https://github.com/npm/npm/issues/16866). I'd rather not commit the lock file until it's extremely clear as to what the flow is (there doesn't appear to be consistency).

Additionally, we may only want to eventually commit to the application version of ProcBots, and not the library. It needs discussion.